views/partials/_whats-new.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="app-whats-new" style="background-color:#f8f8f8; border-bottom:1px solid #b1b4b6">
+  <div class="app-width-container">
+    <div class="govuk-main-wrapper govuk-main-wrapper--l">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full-from-desktop">
+        <h2 id="whats-new" class="govuk-heading-l">What's new</h2>
+        <p class="govuk-body">20 May 2021: We’ve updated guidance in the cookie banner component. <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/12#issuecomment-845097722" class="govuk-link">Read about this update</a>.</p>
+        <p class="govuk-body"><a href="http://eepurl.com/hye639" class="govuk-link">Sign up to get update emails about the Design System</a>.</p>
```

Comment:
Can we use the full url here instead of a shortened url? I think it would be better for users, because it would be clear where they are being sent, and they will not be tracked (as much). [The service manual says something similar about URLs in emails](https://www.gov.uk/service-manual/technology/how-to-email-your-users#protect-your-users), which what made me think of this suggestion.

```suggestion
        <p class="govuk-body"><a href="https://service.us1.list-manage.com/subscribe?u=ada1bae4b177beb9e03502ced&id=367f0a5723" class="govuk-link">Sign up to get update emails about the Design System</a>.</p>
```